v1.16/Gemfile

```diff
@@ -0,0 +1,64 @@
+source 'https://rubygems.org'
+
+gem 'oj', '3.14.2'
+gem 'activesupport', '6.1.7'
+gem 'http_parser.rb', '0.8.0'
+gem 'tzinfo', '>= 1.2.11'
+gem 'resolv', '0.2.2'
+gem 'json', '2.6.3'
+gem 'yajl-ruby', '1.4.1'
+gem 'async', '1.31.0'
+gem 'async-http', '0.60.1'
+gem 'fluentd', '1.16.2'
+gem 'prometheus-client', '4.1.0'
+gem 'bigdecimal', '3.1.4'
+gem 'webrick', '1.8.1'
+gem 'gelf', '3.1.0'
+gem 'elasticsearch', '8.6.0'
+gem 'elasticsearch-xpack', '7.17.7'
+gem 'snappy', '0.2.0'
+gem 'fluent-plugin-detect-exceptions', '0.0.15'
+gem 'fluent-plugin-remote-syslog', '1.1.0'
+gem 'fluent-plugin-webhdfs', '1.5.0'
+gem 'fluent-plugin-prometheus', '2.0.3'
+gem 'fluent-plugin-s3', '1.7.2'
+gem 'fluent-plugin-rewrite-tag-filter', '2.4.0'
+# gem 'fluent-plugin-azure-storage-append-blob', '0.2.1'
+gem 'fluent-plugin-aliyun-oss', '0.0.1'
+gem 'fluent-plugin-dedot_filter', '1.0.0'
+gem 'fluent-plugin-sumologic_output', '1.8.0'
+gem 'fluent-plugin-kafka', '0.18.1'
+gem 'fluent-plugin-geoip', '1.3.2'
+gem 'fluent-plugin-tag-normaliser', '0.1.2'
+gem 'fluent-plugin-grafana-loki', '1.2.20'
+gem 'fluent-plugin-concat', '2.5.0'
+gem 'fluent-plugin-kinesis', '3.4.2'
+gem 'fluent-plugin-parser-logfmt', '0.0.0'
+gem 'fluent-plugin-multi-format-parser', '1.0.0'
+gem 'fluent-plugin-record-modifier', '2.1.1'
+gem 'fluent-plugin-splunk-hec', '1.3.2'
+gem 'fluent-plugin-elasticsearch', '5.3.0'
+gem 'fluent-plugin-newrelic', '1.2.2'
+gem 'fluent-plugin-cloudwatch-logs', '0.14.3'
+gem 'fluent-plugin-opensearch', '1.1.2'
+# gem 'fluent-plugin-aws-elasticsearch-service', '2.4.1'

```

Comment:
dormant since 2021:

https://github.com/atomita/fluent-plugin-aws-elasticsearch-service/tree/master

Reply:
Alternative https://github.com/fluent/fluent-plugin-opensearch?

Reply:
I haven't worked with any of these personally, but I would opt for going with the opensearch plugin if it supersedes the other one.

Reply:
if i remember correctly, there were API differences at least between opendistro for elastic search and opensearch. i assume that applies to the aws elastic service as well.